AndelaOSP/shufflebox-backend
# # 138315199 Add Google Social Login to API

#### What does this PR do?
It adds Google login to the API.

#### Description of Task to be completed?
Add feature allowing API consumers to be authenticated using their Google accounts.
The Andela authentication micro service authenticates a user using their Andela email accounts and sets a token with the JWT token as a browser cookie.
The JWT token should then be included in all the requests to the API in the Authorization header.

#### How should this be manually tested?
Run the API locally `python3 manage.py runserver 8080`.
Click [here](https://api-staging.andela.com/login?redirect_url=http://localhost:8080) to get authenticated by the Andela authentication service. If successful, you will be redirected to `localhost:8080`.
Your browser should have a new cookie with the name jwt-token. Copy the value and use that while making requests to the API.
To include the token in your requests, add an **Authorization** header where the value is preceded by the word JWT like so: `JWT your_token_here`.

#### What are the relevant pivotal tracker stories?
#138315199
